src/resonanceReconstruction/rmatrix/test/fromENDFResolvedReichMoore.test.hpp

```diff
@@ -205,1806 +311,3187 @@ SCENARIO( "fromENDF - LRF3" ) {
+    " 2.442239+3 1.000000+0 4.350887-2 4.083304-2 2.685357-2 0.000000+09437 2151     \n"
+    " 2.446971+3 0.000000+0 1.099686-1 3.999520-2-7.701130+0 1.000000-79437 2151     \n"
+    " 2.448527+3 1.000000+0 7.076491-3 4.264229-2 3.730102-2 0.000000+09437 2151     \n"
+    " 2.450849+3 1.000000+0 6.815714-2 3.985104-2-7.957897-2 0.000000+09437 2151     \n"
+    " 2.456585+3 1.000000+0 2.022294-2 4.290266-2 1.145612-1 0.000000+09437 2151     \n"
+    " 2.460223+3 0.000000+0 1.078125-2 4.406361-2 2.653408-1 1.000000-79437 2151     \n"
+    " 2.462643+3 1.000000+0 2.507513-2 4.084632-2-5.396590-2 0.000000+09437 2151     \n"
+    " 2.466188+3 1.000000+0 4.681939-3 3.025920-2-2.556120-2 0.000000+09437 2151     \n"
+    " 2.469624+3 1.000000+0 3.680925-2 3.981501-2 8.971018-3 0.000000+09437 2151     \n"
+    " 2.471418+3 1.000000+0 4.835053-3 3.641830-2 8.294363-2 0.000000+09437 2151     \n"
+    " 2.473086+3 1.000000+0 8.894406-3 3.534725-2-2.462633-2 0.000000+09437 2151     \n"
+    " 2.476036+3 0.000000+0 6.996022-3 4.300423-2 1.453861+0 1.000000-79437 2151     \n"
+    " 2.477380+3 1.000000+0 2.277438-2 3.543532-2-2.084932-2 0.000000+09437 2151     \n"
+    " 2.480042+3 1.000000+0 5.399367-3 3.204663-2-5.341699-2 0.000000+09437 2151     \n"
+    " 2.482107+3 1.000000+0 2.353772-2 4.530128-2 1.693341-1 0.000000+09437 2151     \n"
+    " 2.483391+3 0.000000+0 1.505215-3 3.899041-2-1.291420+0 1.000000-79437 2151     \n"
+    " 2.484322+3 1.000000+0 4.534185-2 4.024252-2-1.063069-2 0.000000+09437 2151     \n"
+    " 2.485523+3 1.000000+0 4.862761-2 6.168987-2 2.027231-1 0.000000+09437 2151     \n"
+    " 2.492112+3 1.000000+0 1.524686-2 1.587093-2-1.448784-2 0.000000+09437 2151     \n"
+    " 2.494814+3 1.000000+0 5.331384-2 3.178817-2 1.106315-2 0.000000+09437 2151     \n"
+    " 2.497162+3 1.000000+0 9.579957-4 3.351754-2-4.020848-3 0.000000+09437 2151     \n"
+    " 2.499517+3 1.000000+0 4.093699-3 3.262239-2-7.769965-3 0.000000+09437 2151     \n"
+    " 2.504370+3 1.000000+0 3.307642-2 3.927163-2 1.818048-1 0.000000+09437 2151     \n"
+    " 2.506800+3 0.000000+0 3.453994-1 3.927646-2 1.326153+0 9.075218-29437 2151     \n"
+    " 2.509230+3 1.000000+0 9.130709-2 3.926767-2-1.348610-1 0.000000+09437 2151     \n"
+    " 2.511660+3 0.000000+0 3.123558-1 3.926596-2 1.002582+0 1.971943-29437 2151     \n"
+    " 2.514090+3 1.000000+0 1.322666-1 3.926598-2-2.281954-1 0.000000+09437 2151     \n"
+    " 2.516520+3 1.000000+0 1.436018-1 3.926556-2 3.888109-2 0.000000+09437 2151     \n"
+    " 2.518950+3 1.000000+0 1.662716-1 3.926594-2 2.224877-1 0.000000+09437 2151     \n"
+    " 2.521380+3 1.000000+0 1.600835-1 3.926564-2 8.565578-2 0.000000+09437 2151     \n"
+    " 2.523800+3 1.000000+0 1.829061-1 3.926666-2 4.074383-1 0.000000+09437 2151     \n"
+    "                                                                  9437 2  0     \n" );
+}
+
+std::string Pu239LRF3WithNLS4() {
```

Comment:
If you don't think it's worth it, you can leave it as is, but...

It's really silly to have 1000 lines of the real Pu evaluation twice.  Can't you just turn the function into something that takes NLS as an argument?  Or use less than 1043 resonances to do this?

Reply:
Done.